src/ProjectTemplates/BlazorTemplates.Tests/BlazorServerTemplateTest.cs

```diff
@@ -2,37 +2,47 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using System.Net;
-using System.Threading;
+using System.Runtime.InteropServices;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.E2ETesting;
-using Microsoft.AspNetCore.Testing;
-using OpenQA.Selenium;
+using Microsoft.AspNetCore.BrowserTesting;
+using PlaywrightSharp;
+using ProjectTemplates.Tests.Infrastructure;
 using Templates.Test.Helpers;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Templates.Test
 {
-    public class BlazorServerTemplateTest : BrowserTestBase
+    public class BlazorServerTemplateTest
     {
-        public BlazorServerTemplateTest(ProjectFactoryFixture projectFactory, BrowserFixture browserFixture, ITestOutputHelper output) : base(browserFixture, output)
+        public BlazorServerTemplateTest(ProjectFactoryFixture projectFactory, PlaywrightFixture<BlazorServerTemplateTest> fixture, ITestOutputHelper output)
         {
             ProjectFactory = projectFactory;
+            Fixture = fixture;
+            Output = output;
+            BrowserContextInfo = new ContextInformation(TestHelpers.CreateFactory(output));
         }
 
         public ProjectFactoryFixture ProjectFactory { get; set; }
 
+        public PlaywrightFixture<BlazorServerTemplateTest> Fixture { get; }
```

Comment:
Fixture is a general Xunit term, right? Might be best to name this `PlaywrightFixture` or something else to distinguish from other Xunit fixtures.

Reply:
This lives in our E2E test project. 

We can call it PlaywrightFactory or something like that (similar to how we have WebApplicationFactory, which is also used as a fixture), but I'm going to leave it like this for the time being.

Once we start to use this in other areas we can rename/move somewhere else